src/services/projects.js

```diff
@@ -99,11 +100,19 @@ module.exports = class ProjectsHelper {
 					throw new Error('FILE_UPLOADED_FAILED')
 				}
 			} catch (error) {
-				return responses.failureResponse({
-					message: error.message || error,
-					statusCode: httpStatusCode.bad_request,
-					responseCode: 'CLIENT_ERROR',
-				})
+				if (error.name === 'SequelizeDatabaseError' && error.original.code === '22001') {
+					return responses.failureResponse({
```

Comment:
@priyanka-TL we should let them know the key for which this validation is coming.